lib/kele.rb

```diff
@@ -13,4 +15,9 @@ def initialize(email,password)
     @auth_token = @response['auth_token']
     puts "Credentials Approved"
   end
+
+  def get_me
+    @response = self.class.get("#{@baseURL}users/me", headers: { "authorization" => @auth_token })
```

Comment:
Consider making this a regular variable (without the `@`). You'll likely not use it outside of this method.